tools/wpt/browser.py

```diff
@@ -1104,6 +1104,38 @@ def version(self, binary=None, webdriver_binary=None):
         return None
 
 
+class WebKitGTKMiniBrowser(WebKit):
+
+    def find_binary(self, venv_path=None, channel=None):
+        libexecpaths = ["/usr/libexec/webkit2gtk-4.0",
```

Comment:
How often does the "4.0" part of this path change, and what will happen when it does?

Reply:
Alright, sounds like a risk we can accept :)